daliuge-common/setup.py

```diff
@@ -89,6 +89,21 @@ def do_versioning():
     author="ICRAR DIA Group",
     author_email="dfms_prototype@googlegroups.com",
     url="https://github.com/ICRAR/daliuge",
+    classifiers=[
+        "Development Status :: 5 - Production/Stable",
+        "Topic :: System :: Distributed Computing",
+        "Topic :: Scientific/Engineering",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: BSD License",
```

Comment:
DALiuGE is licensed [under LGPL2.1](https://github.com/ICRAR/daliuge/blob/master/LICENSE).

Reply:
Sorry, I originally couldn't decide between
* `License :: OSI Approved :: GNU Lesser General Public License v2 (LGPLv2)`
* `License :: OSI Approved :: GNU Lesser General Public License v2 or later (LGPLv2+)`

It sounds like first one is the best fit? complete list is [on pypi filters](https://pypi.org/search/?q=gpl&o=&c=License+%3A%3A+OSI+Approved+%3A%3A+GNU+Lesser+General+Public+License+v2+%28LGPLv2%29&c=License+%3A%3A+OSI+Approved+%3A%3A+GNU+Lesser+General+Public+License+v2+or+later+%28LGPLv2%2B%29&c=License+%3A%3A+OSI+Approved+%3A%3A+GNU+Library+or+Lesser+General+Public+License+%28LGPL%29)

Reply:
First one sounds good to me.